src/Middleware/CORS/test/FunctionalTests/package.json

```diff
@@ -2,7 +2,7 @@
   "devDependencies": {
     "jest": "^23.6.0",
     "merge": "^1.2.1",
-    "puppeteer": "^1.8.0"
+    "puppeteer": "^1.12.2"
```

Comment:
NPM did this for me.

Reply:
cc @Eilon in case we need to notify LCA.

Reply:
Component Governance will pick it up. Good to go.